backend/app/models/sample_type.rb

```diff
@@ -0,0 +1,284 @@
+# sample_types table
+class SampleType < ActiveRecord::Base
+
+  validates :name,        presence: true, uniqueness: { case_sensitive: false }
+  validates :description, presence: true
+
+  # Return all sample types.
+  #
+  # @return all sample types
+  def self.find_all
+    sql = "
```

Comment:
Please use the 
```<<-SQL 
select * from sample_types order by name
SQL
```
for raw SQL so that editors that support it (VS Code) will do syntax highlighting.